### PR TITLE
Bug 1787660: Add CLI flag to set ClusterOperator name.

### PR DESCRIPTION
**Pull request by benluddy**
If the flag is set to a non-empty string, and config.openshift.io/v1
isn't available, then the operator exits with an error
status. Previously, the absence of configv1 would log a warning and
continue running without ever updating ClusterOperator status. If this
occurred during an OpenShift cluster upgrade, the upgrade would time
out waiting for marketplace to report success via its ClusterOperator.

Outside of an OpenShift cluster, the new flag can be omitted (or set
to the empty string) to disable ClusterOperator status updates
altogether.

### CODE REVIEW

File: pkg/operatorsource/handler.go [hunk 1]
```diff
@@ -4,7 +4,7 @@ import (
 	"context"
 
 	"github.com/operator-framework/operator-marketplace/pkg/apis/operators/shared"
-	"github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
+	v1 "github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
```
Comment (kevinrizza) [inline pkg/operatorsource/handler.go]: What's the point of this alias?

Comment (benluddy) [reply pkg/operatorsource/handler.go]: That is weird. I have an after-save hook to tell `gopls` to organize imports, but I didn't notice that it had done that.

**Pull request merged**